login_server/src/cpp/ServerConfig.cpp

```diff
@@ -57,6 +57,7 @@ namespace ServerConfig {
 	int         g_FakeLoginSleepTime = 820;
 	std::string g_versionString = "";
 	bool		g_disableEmail = false;
+	bool	    g_resendTransactionOnStart = false;
```

Comment:
I suggest to have the name equal to the config value just to avoid confusion.

```suggestion
	bool	    g_resendUnfinishedTransactionOnStart = false;
```

Reply:
Ich wollte nicht so einen langen Variablennamen haben, aber du hast recht.